src/MongoDbOperationStore.ts

```diff
@@ -54,7 +56,14 @@ export default class MongoDbOperationStore implements OperationStore {
       batch.insert(mongoOperation);
     }
 
-    await batch.execute();
+    try {
+      await batch.execute();
+    } catch (e) {
```

Comment:
Nit: `error` instead of `e` (group decision).

Reply:
Done.